orttraining/orttraining/test/training_ops/cuda/layer_norm_test.cc

```diff
@@ -56,13 +59,19 @@ static void TestLayerNormGrad(
     EigenRowVectorArrayMap inv_std_var{inv_std_var_data.data(), N};
 
     mean = X.colwise().mean();
```

Comment:
need not compute mean for simplifiedLN

Reply:
Moved mean computation into old layer norm branch